src/components/OptionsList.js

```diff
@@ -77,6 +80,7 @@ const defaultProps = {
     optionBackgroundColor: undefined,
     optionHoveredStyle: undefined,
     contentContainerStyles: [],
+    listContainerStyles: [],
```

Comment:
Oh I see the issue now, if you give this a default value of `undefined` the solution will work because below you are checking:

```js
this.props.listContainerStyles ? this.props.listContainerStyles : [styles.flex1]
```

and empty array will be truthy value